src/MoBi.Presentation/Serialization/Xml/SettingsPersistor.cs

```diff
@@ -36,11 +36,23 @@ public virtual void Save()
          {
             var serializer = _serializerRepository.SerializerFor(_settings);
             var xml = serializer.Serialize(_settings, serializationContext);
-            var doc = XDocument.Load(new StringReader(xml.ToString()));
-            doc.Save(SettingsFilePath);
+            saveSerializedSettingsToFile(xml);
          }
       }
 
+      private void saveSerializedSettingsToFile(XElement xml)
```

Comment:
Just ensuring that the folder exists before saving. This is something that does not happen when installed with setup because folder is already created. 
There is absolutely no test on this behavior. This should be added when we have time for that